apps/marketplace/components/Brief/BriefResponseForm.js

```diff
@@ -0,0 +1,179 @@
+              Apply for &lsquo;{brief.title}&rsquo;
+            </h1>
+          </header>
+
+          <Form model={model} id="briefResponse" onSubmit={data => handleSubmit(data)}>
+            <h2>Do you have the essential skills and experience?</h2>
+            <p>You must have all essential skills and experience to apply for this opportunity.</p>
+            {brief.essentialRequirements &&
+              brief.essentialRequirements.map((requirement, i) =>
+                <Textarea
+                  key={`essentialRequirement.${i}`}
+                  model={`${model}.essentialRequirements[${i}]`}
+                  name={`essentialRequirement.${i}`}
+                  id={`essentialRequirement.${i}`}
+                  controlProps={{ limit: 150 }}
+                  label={requirement}
+                  validators={{ required }}
+                  showMessagesDuringFocus
+                  messages={{
+                    required: `Essential skills and experience are required`
+                  }}
+                />
+              )}
+
+            <h2>Do you have any of the nice-to-have skills and experience?</h2>
+            <p>This section is optional but may help your application stand out.</p>
+            {brief.niceToHaveRequirements &&
+              brief.niceToHaveRequirements.map((requirement, i) =>
+                <Textarea
+                  key={`niceToHaveRequirement.${i}`}
+                  model={`${model}.niceToHaveRequirements[${i}]`}
+                  name={`niceToHaveRequirement.${i}`}
+                  id={`niceToHaveRequirement.${i}`}
+                  controlProps={{ limit: 150 }}
+                  label={requirement}
+                />
+              )}
+            <Textfield
+              model={`${model}.availability`}
+              name="availability"
+              id="availability"
+              htmlFor="availability"
+              label="When can you start?"
+              description="For example: 31/12/2016"
+              validators={{
+                required
+              }}
+              messages={{
+                required: 'Availability is required'
+              }}
+            />
+            <Textfield
+              model={`${model}.respondToEmailAddress`}
+              name="respondToEmailAddress"
+              id="respondToEmailAddress"
+              htmlFor="respondToEmailAddress"
+              label="Contact email"
+              description="All communication about your application will be sent to this address."
+              validators={{
+                required,
+                validEmail
+              }}
+              messages={{
+                required: 'A contact email is required',
+                validEmail: 'A valid contact email is required'
+              }}
+            />
+            {supplierCode &&
+              brief.lotSlug &&
+              brief.lotSlug === 'digital-professionals' &&
+              <span>
+                <Textfield
```

Comment:
I typed `$100` here - client didn't complain but server returned `not_money_format` (which was not displayed in error box)